main.tsx

```diff
@@ -70,20 +69,17 @@ if (import.meta.main) {
       owner: "thefrontside",
       name: "effection-contrib",
     });
-    yield* ContribRepositoryContext.set(contrib);
 
-    let docs = yield* loadDocs({ repo: library, pattern: "effection-v3" });
-    let docsV4 = yield* loadDocs({ repo: library, pattern: "effection-v4" });
+    yield* ContribRepositoryContext.set(contrib);
 
     let revolution = createRevolution({
       app: [
         route("/", indexRoute()),
         route("/search", searchRoute()),
         route(
-          "/docs/v4/:id",
-          docsRoute({ docs: docsV4, search: true, series: "v4" }),
+          "/docs{/:series}{/:id}",
+          guidesRoute({ repository: library, search: true }),
```

Comment:
While I vastly prefer the path-to-regexp v8 syntax for optional parameters, I think it's actually cleaner to explicitly map the redirects in the router, rather than have a set of nullable params that mean you have to jump through some hidden if statements hidden in the route handler:

```ts
route("/docs/:series", redirectToSeriesIndex()),
route("/docs/:series/:id", guidesRoute(....),
```

That way each route is guaranteed to have the parameters it needs.

Reply:
Yeah, I was thinking the same thing.

Reply:
Also, we need a strategy for handling redirects in the static version of our site and how to integrate that with staticalize.

Maybe something like a `/.redirects` file in the root?

Reply:
Yeah, we need something that would allow us to handle redirects in the server. In the meantime, I created a soft redirect for docs -> guides. This should be ready to go.